subhub/app.py

```diff
@@ -13,7 +13,7 @@
 
 
 def create_app(config=None):
-    if not CFG("AWS_EXECUTION_ENV", None):
+    if not CFG.AWS_EXECUTION_ENV:
         options = {"swagger_ui": True}
         region = "localhost"
         host = "http://localhost:8000"
```

Comment:
FWIW, looks like `DYNALITE_PORT` should get used here too. Had to make that tweak locally to get this working.

Reply:
oh good catch!